packages/openapi3/src/lib.ts

```diff
@@ -1,6 +1,7 @@
 import { createTypeSpecLibrary, JSONSchemaType, paramMessage } from "@typespec/compiler";
 
 export type FileType = "yaml" | "json";
+export type OutputSpecVersions = "v3.0" | "v3.1";
```

Comment:
For when this option make it if not now, I think should maybe just use the full version you'd see in the document without the `v`

The name maybe also should be `openapi-version`?

Reply:
Sure - so make these `3.0.0` and `3.1.0` instead. Will update.